system/mrm_handler/README.md

```diff
@@ -0,0 +1,44 @@
+# mrm_handler
+
+## Purpose
+
+MRM Handler is a node to select proper MRM from from system failure state contained in OperationModeAvailability.
```

Comment:
MRM Handler is a node to select **a** proper MRM from **a** system failure state

Reply:
Thanks for your comment！